lib/ui/hooks.dart

```diff
@@ -288,8 +288,7 @@ bool _onError(Object error, StackTrace? stackTrace) {
   return PlatformDispatcher.instance._dispatchError(error, stackTrace ?? StackTrace.empty);
 }
 
-// ignore: always_declare_return_types, prefer_generic_function_type_aliases
-typedef _ListStringArgFunction(List<String> args);
+typedef _ListStringArgFunction = Object? Function(List<String> args);
```

Comment:
Whoa, we still have non-assignment typedefs in the code?

Reply:
Might be good to search through for other ignores of `prefer_generic_function_type_aliases`!